SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1667,8 +1667,37 @@ void USpatialNetDriver::ProcessRemoteFunction(
 	}
 }
 
+void USpatialNetDriver::PollPendingLoads()
+{
+	if (PackageMap)
+	{
+		for (USpatialPackageMapClient::PendingRefSet::TIterator IterPending = PackageMap->PendingReferences.CreateIterator(); IterPending; ++IterPending)
```

Comment:
use `auto`

Reply:
I need to have the iterator to call RemoveCurrent.

Reply:
@ImprobableNic I think Sami means to use `auto IterPending = PackageMap->PendingReferences.CreateIterator()`

Reply:
Oh, right.